modules/iam/groups/main.tf

```diff
@@ -0,0 +1,43 @@
+// Define groups
+# trivy:ignore:AVD-AWS-0123
```

Comment:
Can we add a comment explaining why we're skipping rules here and elsewhere? It's simple to dismiss all warnings, but a better approach is to address them and, if not possible, explain why we've set them aside

Reply:
Added explaination in [216c662](https://github.com/nvminhtue/tyson_devops_ic/pull/23/commits/216c6628bd63c3a616b795c11beae32c41478579)